cdap-docs/developers-manual/source/getting-started/dev-env.rst

```diff
@@ -49,11 +48,41 @@ Using IntelliJ
 
 Using Eclipse
 -------------
-
 1. In your `Eclipse <https://www.eclipse.org/>`__ installation, make sure you have the
    `m2eclipse <http://m2eclipse.sonatype.org>`__ plugin installed.
 #. Go to menu *File -> Import*
 #. Enter *maven* in the *Select an import source* dialog to filter for Maven options.
 #. Select *Existing Maven Projects* as the import source.
 #. Browse for the Maven project's directory.
 #. Click *Finish*, and the new CDAP project will be imported, created and opened.
+
+Running CDAP from within an IDE
+-------------------------------
+As CDAP is an open source project, you can download the source, import it into an IDE,
+then modify, build, and run CDAP.
+
+To do so, follow these steps:
+
+1. Install all the :ref:`prerequisite system requirements <system-requirements>` for CDAP development.
+#. Download the source as a ZIP from |source-link|.
```

Comment:
I don't see the page you pointed to has any mention of "zip". Do I misunderstand something??